.circleci/config.yml

```diff
@@ -84,10 +86,11 @@ jobs:
       - run: sudo apt-get update
       - run: sudo apt-get install tree
       - run: sudo apt-get install cf-cli
+      - run: cf install-plugin https://github.com/govau/autopilot/releases/download/0.0.5-venapp/autopilot-linux -f
```

Comment:
So is Autopilot the standard way for us to be deploying now? (I hope so, as it's a lot less hassle than fiddly green-blue setups)

Reply:
(By 'us' I mean anyone at DTA deploying to Cloud... sorry to butt in on this PR, I just noticed it in my email :)

Reply:
I would recommend it - the gotcha is that your app manifest must fully describe your app, so any env variables that aren't in the manifest are zapped on a new deploy.

We're working through some techniques with marketplace right now to do this and will write up in detail when done.

Reply:
@aeijdenberg we should think about updating  https://docs.cloud.gov.au/deployment_configuration/zero_downtime_deploys/